content/influxdb/v2.0/reference/api/influxdb-1x/query.md

```diff
@@ -81,33 +81,33 @@ The following precisions are available:
 
 ##### Query using basic authentication
 ```sh
-curl -G https://cloud2.influxdata.com/query \
-  -u username:YourAuthToken \
+curl --request GET https://cloud2.influxdata.com/query \
+  --header "username:YourAuthToken" \
```

Comment:
This flag should be `--user`. It's using curl's basic auth functionality rather than passing an authorization header.